modules/ROOT/pages/activity-tracking.adoc

```diff
@@ -23,15 +23,19 @@ Select a transmission to view its details and view and download the message cont
 +
 You can retrieve up to one week's worth of messages.
 +
-* Acknowledgement status, such as `Overdue`
 * Message processing status, such as `Received`
+* Acknowledgement status, such as `Overdue`
 * Partner message type, such as `X12 - 850`
 * Host message type, such as `JSON host-purchase-order`
+* Transmission ID
 +
 Partner Manager returns a list of transmissions that meet your search criteria along with related information, such as the date the transmission was received, number of messages in the transmission, message type, control number, and acknowledgement status.
 +
-. Click on a transmission to review additional information about the transmission and to view its message flow.
+. Do either of the following:
+** Click on a transmission to review additional information about the transmission and to view its message flow.
+** Click on the *messages* link to access the message view, where you can see information about the messages in the transaction, as described in xref:activity-message-tracking.adoc[Activity Message Tracking].
```

Comment:
```suggestion
** Click *messages* to access the message view to see information about the messages in the transaction, as described in xref:activity-message-tracking.adoc[Activity Message Tracking].
```

Reply:
https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-%22click%22

Reply:
(do you need "to access the message view"?)